### PR TITLE
Initialize auro-nav component

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #1 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Create the `auro-nav` component with examples and documentation.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
I'll be adding individual comments for various items that need changed.

**Comment by jason-capsule42** [comment]
`./docs/partials/useCases.md` needs to be completed.

**Comment by jason-capsule42** [comment]
`/docs/partials/desription.md` needs to be completed.

**Comment by jason-capsule42** [comment]
This was committed without a build run so several compiled files were not updated:

readme.md
docs/api.md
demo/apiExamples.md
demo/demo.md

**Comment by jason-capsule42** [comment]
There are *many* references to `auro-breadcrumb` throughout the repo that need to be updated to `auro-nav`.

**Comment by jason-capsule42** [comment]
Build is failing on the rollup command.

**Comment by jason-capsule42** [comment]
Test file is still named `auro-breadcrumb.test.js`.

File: docs/partials/useCases.md [hunk 1]
```diff
@@ -1,5 +1 @@
-The `<auro-breadcrumb>` element should be used in situations where users may:
-
-* ...
-* ...
-* ...
+The `<auro-nav>` element acts as a container for a series of `<auro-hyperlinks>`. The `<auro-nav>` element should only be used for multi-page projects that have 3 to 5 levels of page depth. Additionally, `<auro-nav>` should only display in the UI on page depth 3 or above.
```
Comment (jason-capsule42) [inline docs/partials/useCases.md:1]:
```suggestion
The `<auro-nav>` element acts as a container for a series of `<auro-hyperlinks>`. The `<auro-nav>` element should only be used for multi-page projects that have 3 to 5 levels of page depth.
```

The second sentence already states what the third sentence repeats. Less words == more understanding.

File: docs/partials/demo.md [hunk 2]
```diff
@@ -1,54 +1,43 @@
...
+## Example(s)
 
-|convallis|tristique|nisl dignissim|eleifend|
-|---|---|---|---|
-|√|√|||
-|||√|√|
+### Default
 
-Aenean at blandit lorem. Fusce imperdiet mi nec gravida maximus. Quisque nisl libero, condimentum in nisi a, imperdiet lacinia arcu.
+The default component supports a `type=breadcrumb` structure, where chevrons are displayed as the divider between each `<auro-hyperlink>`.
```
Comment (jason-capsule42) [inline docs/partials/demo.md:15]: I think we need to punt the default example because there is no working version of just `<auro-nav></auro-nav>` we currently only support `<auro-nav type="breadcrumb"></auro-nav>`

The doc needs a section for `### Type` and then subsection for `####  Breadcrumb".

This should end up on both the demo page and the API page.

File: package.json [hunk 3]
```diff
@@ -6,26 +6,30 @@
...
-    "url": "https://github.com/AlaskaAirlines/auro-breadcrumb"
+    "url": "https://github.com/AlaskaAirlines/auro-nav"
   },
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
     "node": ">=16.13.0"
   },
   "dependencies": {
+    "@alaskaairux/icons": "^4.19.0",
```
Comment (jason-capsule42) [inline package.json:22]: This is already a peer dependency and probably shouldn't be here.

File: package.json [hunk 4]
```diff
@@ -6,26 +6,30 @@
...
+    "@aurodesignsystem/auro-input": "^2.10.3",
```
Comment (jason-capsule42) [inline package.json:23]: Why is this necessary?

File: package.json [hunk 5]
```diff
@@ -6,26 +6,30 @@
...
+    "chalk": "^4.1.2",
     "lit-element": "^2.5.1",
-    "lit-html": "^1.4.1",
-    "chalk": "^4.1.2"
+    "lit-html": "^1.4.1"
```
Comment (jason-capsule42) [inline package.json:26]: Let's build this using the latest version of Lit. You can find Lit update instructions via google.

File: test/auro-nav.test.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import { fixture, html, expect } from '@open-wc/testing';
+import '../src/auro-nav.js';
+
+describe('auro-nav', () => {
+  it('auro-nav is accessible', async () => {
```
Comment (jason-capsule42) [inline test/auro-nav.test.js:5]: Should have a test for `type="breadcrumb"`

File: src/auro-nav.js [hunk 7]
```diff
@@ -55,16 +54,14 @@ class AuroBreadcrumb extends LitElement {
...
       </div>
     `;
   }
 }
 
 // define the name of the custom component
-if (!customElements.get("auro-breadcrumb")) {
-  customElements.define("auro-breadcrumb", AuroBreadcrumb);
+if (!customElements.get("auro-nav")) {
+  customElements.define("auro-nav", Auronav);
```
Comment (jason-capsule42) [inline src/auro-nav.js:66]:
```suggestion
  customElements.define("auro-nav", AuroNav);
```

File: src/auro-nav.js [hunk 8]
```diff
@@ -16,16 +16,18 @@ import "focus-visible/dist/focus-visible.min.js";
...
  *
- * @attr {Boolean} fixed - Uses fixed pixel values for element shape
- * @attr {String} cssClass - Applies designated CSS class to demo element - you want to delete me!
+ * @attr {String} type - Populates the `type` attribute on the nav. Allowed values are `breadcrumb`.
+ * @slot Slot for insertion of auro-hyperlinks.
  */
 
 // build the component class
-class AuroBreadcrumb extends LitElement {
+class Auronav extends LitElement {
```
Comment (jason-capsule42) [inline src/auro-nav.js:30]:
```suggestion
class AuroNav extends LitElement {
```

File: rollup.config.js [hunk 9]
```diff
@@ -53,13 +53,13 @@ const modernConfig = {
   ]
 };
 
-const auroBreadcrumbConfig = {
+const auronavConfig = {
```
Comment (jason-capsule42) [inline rollup.config.js:56]:
```suggestion
const auroNavConfig = {
```

File: rollup.config.js [hunk 10]
```diff
@@ -53,13 +53,13 @@ const modernConfig = {
...
   output: {
     format: 'iife',
-    file: 'dist/auro-breadcrumb__bundled.es5.js'
+    file: 'dist/auro-nav__bundled.es5.js'
   },
   plugins: getSharedPlugins(true)
 };
 
-export default [modernConfig, auroBreadcrumbConfig];
+export default [modernConfig, auronavConfig];
```
Comment (jason-capsule42) [inline rollup.config.js:65]:
```suggestion
export default [modernConfig, auroNavConfig];
```

File: package.json [hunk 11]
```diff
@@ -74,11 +78,10 @@
     "postcss-remove-rules": "^1.0.0",
     "postcss-selector-replace": "^1.0.2",
     "prismjs": "^1.29.0",
-    "rollup": "^3.7.5",
+    "rollup": "^2.70.1",
```
Comment (jason-capsule42) [inline package.json:81]: Why is this a change to an earlier version?

File: package.json [hunk 12]
```diff
@@ -74,11 +78,10 @@
...
     "rollup-plugin-minify-html-literals": "^1.2.6",
-    "rollup-plugin-serve": "^2.0.2",
+    "rollup-plugin-serve": "^1.1.0",
```
Comment (jason-capsule42) [inline package.json:83]: Why is this a change to an earlier version?

File: package.json [hunk 13]
```diff
@@ -74,11 +78,10 @@
...
     "rollup-plugin-terser": "^7.0.2",
-    "semantic-release": "^19.0.5",
```
Comment (jason-capsule42) [inline package.json:81]: Why was this removed?

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: package.json (hunk 11 above)
Comment (jordanjones243) [reply package.json:78]: There is an issue running the build with the older versions of rollup.

File: package.json (hunk 12 above)
Comment (jordanjones243) [reply package.json:80]: There is an issue running the build with the older versions of rollup.

**Review by jason-capsule42 — DISMISSED** [review]
Screenshot 2023-01-31 at 5 46 10 PM

I am unable to run the tests on a clean clone of the repo branch.

File: docs/partials/apiExamples.md [hunk 14]
```diff
@@ -3,104 +3,20 @@
...
-### Property Examples
+##### Breadcrumb
 
-#### `propertyName`
-
-Explanation and use description.
+The default example when the `type=breadcrumb` attribute is present.
 
 <div class="exampleWrapper">
   <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
```
Comment (jason-capsule42) [inline docs/partials/apiExamples.md:16]:
```suggestion
  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/breadcrumb.html) -->
```

Let's rename the example file(s) for this one.

File: apiExamples/basic.html [hunk 15]
```diff
@@ -1 +1,7 @@
-<auro-breadcrumb>Hello World</auro-breadcrumb>
+<auro-nav>
+  <auro-nav-breadcrumb href="#">Home</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="#">Level 2</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="#">Level 3</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="#">Level 4</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb>Current</auro-nav-breadcrumb>
```
Comment (jason-capsule42) [inline apiExamples/basic.html:6]: The href causes the user to navigate to somewhere odd. let's make these go to the default aur-nav url on the docsite.
```suggestion
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Home</auro-nav-breadcrumb>
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 2</auro-nav-breadcrumb>
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 3</auro-nav-breadcrumb>
  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 4</auro-nav-breadcrumb>
  <auro-nav-breadcrumb>Current</auro-nav-breadcrumb>
```

File: docs/partials/apiExamples.md [hunk 16]
```diff
@@ -3,104 +3,20 @@
...
+#### Type
+The `type` attribute determines the type of divider to be used to seperate all of the `auro-hyperlinks`.
 
-### Property Examples
+##### Breadcrumb
 
-#### `propertyName`
-
-Explanation and use description.
+The default example when the `type=breadcrumb` attribute is present.
```
Comment (jason-capsule42) [inline docs/partials/apiExamples.md:13]:
```suggestion
Example when `auro-nav` slot is populated with `auro-nav-breadcrumb` elements.
```

Let's also add language explaining that the last breadcrumb should be the current page and that they should *not* populate the HREF attribute.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style-breadcrumb.scss [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+    width: 24px;
+    height: 24px;
+
+    top: -1px;
+    line-height: 24px;
+
+    margin-right: 5px;
+    margin-left: var(--auro-size-100);
+
+    background-color: var(--auro-color-text-secondary-on-light);
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/style-breadcrumb.scss:43]: Where are you seeing this? From the figma, I see that the breadcrumb dividers (chevron-right) are grey (`--auro-color-secondary-on-light`), not black (`--auro-color-primary-on-light`).

[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/style-breadcrumb.scss:43]: I am basing this off of what color I see visually because I cannot see a color requirement in the file.

[the remark this answers is not in the document]
Comment (jason-capsule42) [reply src/style-breadcrumb.scss:43]: This is what June gave me:

https://www.figma.com/file/OJoHbhCPRr8P6DEAvSZyL4/Breadcrumb?node-id=322%3A298&t=f0esyQ2RbnzebPbl-0

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style-breadcrumb.scss (hunk 17 above)
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/style-breadcrumb.scss:43]: After speaking to June, the correct color to use is `--auro-color-icon-primary-on-light`

File: src/style-breadcrumb.scss [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+    content: '';
+
+    mask-image: url('https://unpkg.com/@alaskaairux/icons@latest/dist/icons/interface/chevron-right.svg');
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/style-breadcrumb.scss:46]: We need additional rules. When trying to fix this now, I referred back to this ticket, where we had the same issue:
- https://github.com/AlaskaAirlines/auro-calendar/pull/38

When trying to use this same fix, it presented an error in the UI. When using both `mask-image` and `webkit-mask-box-image` in this circumstance, the chevrons looked to appear darker and thicker than if we just used one rule. What are your thoughts?

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
There are still a few unresolved comments from the previous review.

**Review by leeejune — DISMISSED** [review]

File: package.json [hunk 19]
```diff
@@ -6,25 +6,26 @@
...
     "type": "git",
-    "url": "https://github.com/AlaskaAirlines/auro-breadcrumb"
+    "url": "https://github.com/AlaskaAirlines/auro-nav"
   },
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
     "node": ">=16.13.0"
   },
   "dependencies": {
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply package.json:21]: All non breaking-change dependencies were updated.

**Review by leeejune — DISMISSED** [review]

File: src/auro-nav.js [hunk 20]
```diff
@@ -0,0 +1,120 @@
...
+    icon.style.lineHeight = '24px';
+    icon.style.position = 'relative';
+    icon.style.top = '-2px';
+    icon.style.marginRight = '4px';
+
+    link.insertAdjacentElement('afterbegin', icon);
+  }
+
+  // When using auroElement, use the following attribute and function when hiding content from screen readers.
+  // aria-hidden="${this.hideAudible(this.hiddenAudible)}"
```
Comment (jason-capsule42) [inline src/auro-nav.js:110]: Just noticed this in the code. Does this need to be in here still?

File: test/auro-nav.test.js [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+    const el = await fixture(html`
+      <auro-nav cssclass="testClass"></auro-nav>
+    `);
+
+    await expect(el).to.be.accessible();
+  });
+
+  it('auro-nav custom element is defined', async () => {
```
Comment (jason-capsule42) [inline test/auro-nav.test.js:13]:
```suggestion
  it('auro-nav custom element is defined using breadcrumb links', async () => {
```

let's make the test unique so that it still makes sense when we add new styles later

File: test/auro-nav.test.js [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+    const el = await !!customElements.get("auro-nav");
+
+    await expect(el).to.be.true;
+  });
+
+  it('icons are present on component', async () => {
```
Comment (jason-capsule42) [inline test/auro-nav.test.js:19]:
```suggestion
  it('breadcrumb icons are present on component', async () => {
```

let's make the test unique so that it still makes sense when we add new styles later

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/auro-nav.js [hunk 23]
```diff
@@ -0,0 +1,117 @@
...
+ * @slot Slot for insertion of auro-hyperlinks.
+ */
+
+// build the component class
+export class AuroNav extends LitElement {
+  constructor() {
+    super();
+
+    /* eslint-disable no-unused-expressions */
+    customElements.get('auro-nav-breadcrumb') || customElements.define('auro-nav-breadcrumb', class extends AuroHyperlink {
```
Comment (blackfalcon) [inline src/auro-nav.js:39]:
```suggestion
    customElements.get('auro-breadcrumb') || customElements.define('auro-breadcrumb', class extends AuroHyperlink {
```

The name `auro-nav-breadcrumb` does not compliant with [Auro's naming convention](https://github.com/AlaskaAirlines/WC-Generator/blob/master/bin/generate.js#L73) guidelines.

File: src/auro-nav.js [hunk 24]
```diff
@@ -0,0 +1,117 @@
...
+      styleCssFixed
+    ];
+  }
+
+  /**
+   * @private
+   * @returns {void} Inserts home and chevron-left icons for every breadcrumb.
+   */
+  manageIcons() {
+    const breadcrumbs = [...this.querySelectorAll('auro-nav-breadcrumb')];
```
Comment (blackfalcon) [inline src/auro-nav.js:72]:
```suggestion
    const breadcrumbs = [...this.querySelectorAll('auro-breadcrumb')];
```

File: demo/api.html [hunk 25]
```diff
@@ -0,0 +1,50 @@
...
+          document.querySelector('main').innerHTML = rawHtml;
+          Prism.highlightAll();
+        });
+    </script>
+    <script type="module" src="../src/auro-nav.js"></script>
+
+    <script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion__bundled.js" type="module"></script>
+    <script src="https://unpkg.com/@alaskaairux/auro-icon@latest/dist/auro-icon__bundled.js" type="module"></script>
+    <script src="https://unpkg.com/@aurodesignsystem/auro-hyperlink@latest/dist/auro-hyperlink__bundled.js" type="module"></script>
+    <script src="https://unpkg.com/@alaskaairux/auro-button@latest/dist/auro-button__bundled.js" type="module"></script>
```
Comment (blackfalcon) [inline demo/api.html:48]: Hyperlink is not needed. This is installed at the component level. 

I am not sure what we need button for?

File: apiExamples/breadcrumb.html [hunk 26]
```diff
@@ -0,0 +1,7 @@
+<auro-nav>
+  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Home</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 2</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 3</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 4</auro-nav-breadcrumb>
+  <auro-nav-breadcrumb>Current</auro-nav-breadcrumb>
```
Comment (blackfalcon) [inline apiExamples/breadcrumb.html:6]:
```suggestion
  <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Home</auro-breadcrumb>
  <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 2</auro-breadcrumb>
  <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 3</auro-breadcrumb>
  <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 4</auro-breadcrumb>
  <auro-breadcrumb>Current</auro-breadcrumb>
```

File: src/auro-nav.js [hunk 27]
```diff
@@ -0,0 +1,117 @@
...
+
+    if (breadcrumbs.length > 0) {
+      breadcrumbs.forEach((breadcrumb) => {
+        this.insertIcon(breadcrumb, 'interface', 'home-stroke', '16px');
+        this.insertIcon(breadcrumb, 'interface', 'chevron-left', '24px');
```
Comment (blackfalcon) [inline src/auro-nav.js:77]:
```suggestion
        this.insertIcon(breadcrumb, 'interface', 'home-stroke', 'var(--auro-size-200)');
        this.insertIcon(breadcrumb, 'interface', 'chevron-left', 'var(--auro-size-300)');
```

File: src/auro-nav.js [hunk 28]
```diff
@@ -0,0 +1,117 @@
...
+
+    icon.setAttribute('customSize', true);
+    icon.setAttribute('customColor', true);
+
+    icon.category = category;
+    icon.name = name;
+
+    icon.style.width = size;
+    icon.style.height = size;
+    icon.style.lineHeight = '24px';
```
Comment (blackfalcon) [inline src/auro-nav.js:101]:
```suggestion
    icon.style.lineHeight = 'var(--auro-size-300)';
```

File: src/auro-nav.js [hunk 29]
```diff
@@ -0,0 +1,117 @@
...
+    icon.style.position = 'relative';
+    icon.style.top = '-2px';
+    icon.style.marginRight = '4px';
```
Comment (blackfalcon) [inline src/auro-nav.js:104]:
```suggestion
    icon.style.marginRight = 'var(--auro-size-50)';
```

File: src/style-breadcrumb.scss [hunk 30]
```diff
@@ -0,0 +1,95 @@
...
+    display: none;
+  }
+
+  // Inserts the right chevron icon after each breadcrumb
+  &:after {
+    position: relative;
+    display: inline-block;
+
+    width: 24px;
+    height: 24px;
```
Comment (blackfalcon) [inline src/style-breadcrumb.scss:35]:
```suggestion
    width: var(--auro-size-300);
    height: var(--auro-size-300);
```

File: src/style-breadcrumb.scss [hunk 31]
```diff
@@ -0,0 +1,95 @@
...
+
+    top: -1px;
+    line-height: 24px;
```
Comment (blackfalcon) [inline src/style-breadcrumb.scss:38]:
```suggestion
    line-height: var(--auro-size-300);
```

File: src/style-breadcrumb.scss [hunk 32]
```diff
@@ -0,0 +1,95 @@
...
+:host(:last-of-type) {
+  color: var(--auro-color-text-secondary-on-light);
+
+  // Hides last breadcrumb divider
+  &:after {
+    content: unset;
+  }
+}
+
+@media (max-width: 768px) {
```
Comment (blackfalcon) [inline src/style-breadcrumb.scss:74]: This is not a breakpoint spec. Where did this come from? 

See mixin support here - https://alaskaairlines.github.io/WebCoreStyleSheets/#mixin-auro_breakpoint--sm

Token name | Value 
-- | --
var(--auro-breakpoint-sm) | 660px 
var(--auro-breakpoint-md) | 1024px 
var(--auro-breakpoint-lg) | 1232px

File: test/auro-nav.test.js [hunk 33]
```diff
@@ -0,0 +1,44 @@
...
+
+async function defaultBreadcrumbFixture() {
+  return await fixture(html`
+    <auro-nav>
+      <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Home</auro-nav-breadcrumb>
+      <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 2</auro-nav-breadcrumb>
+      <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 3</auro-nav-breadcrumb>
+      <auro-nav-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 4</auro-nav-breadcrumb>
+      <auro-nav-breadcrumb>Current</auro-nav-breadcrumb>
+    </auro-nav>
```
Comment (blackfalcon) [inline test/auro-nav.test.js:42]:
```suggestion
    <auro-nav>
      <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Home</auro-breadcrumb>
      <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 2</auro-breadcrumb>
      <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 3</auro-breadcrumb>
      <auro-breadcrumb href="http://auro.alaskaair.com/components/auro/nav">Level 4</auro-breadcrumb>
      <auro-breadcrumb>Current</auro-breadcrumb>
    </auro-nav>
```

File: .github/ISSUE_TEMPLATE/blueprint.md [hunk 34]
```diff
@@ -2,7 +2,7 @@
 name: Component Blueprint
 about: Checklist for releasing a new component blueprint
 title: '[Component Blueprint]'
-labels: 'auro-breadcrumb, Type: Design, not-reviewed'
+labels: 'auro-nav, Type: Design, not-reviewed'
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/blueprint.md:5]: Please just delete this file. Not needed.

File: .github/ISSUE_TEMPLATE/publish_component.md [hunk 35]
```diff
@@ -2,7 +2,7 @@
 name: Publish Component
 about: Create a ticket to drive compliance before publishing a new component to Auro Design System
 title: 'Publish to Auro Design System'
-labels: 'auro-breadcrumb, not-reviewed'
+labels: 'auro-nav, not-reviewed'
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/publish_component.md:5]: Please delete this file. Not needed.

File: src/auro-nav.js (hunk 23 above)
Comment (blackfalcon) [reply src/auro-nav.js:39]: This is an interesting new concept. I want to discuss this as a group to ensure we are not simply using a shiny new tool and that there is a plan for expansion. E.g. what if there is a new 'nav' element? How is that going to be supported? 

What is the real value of `auro-nav`? To create `auro-breadcrumb` on the fly? Then `auro-breadcrumb` cannot be used outside of `auro-nav`?

**Review by blackfalcon — DISMISSED** [review]

File: apiExamples/breadcrumb.html (hunk 26 above)
Comment (blackfalcon) [reply apiExamples/breadcrumb.html:6]: Include examples that illustrate the use of `target="_blank"` for both internal link and external.

**Comment by jordanjones243** [comment]
This ticket is blocked until the Auro team discusses the current architecture of the component and whether or not there should be a refactor.

**Review by jason-capsule42 — APPROVED** [review]

**Review by jason-capsule42 — APPROVED** [review]
